magellan-library/src/test/java/com/wealthfront/magellan/ScreenTest.java

```diff
@@ -149,20 +148,4 @@ public void whenTransitionFinished_afterTransitionFinished() {
     screen.whenTransitionFinished(listener);
     verify(listener).onTransitionFinished();
   }
-
-  private static class DummyScreen extends Screen<BaseScreenView<DummyScreen>> {
```

Comment:
nice pulling this out for re-use. Minor: looks like the same code exists in ScreenGroupTest.java if you're willing to pull it out of there as well?

Reply:
d